code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -71,7 +71,7 @@
 	name = "clown shoes"
 	icon_state = "clown"
 	item_state = "clown_shoes"
-	slowdown = SHOES_SLOWDOWN+1
+	slowdown = 1
```

Comment:
Doesn't this add more of a slowdown than there was before? `SHOES_SLOWDOWN` was -1.

Reply:
We are essentially moving `SHOES_SLOWDOWN` into `HUMAN_DELAY` in the configs, so overall there is no change to this item. It will still increase the delay by `1` just as it does here relative to normal shoes in current code.